modules/serverless-tech-preview-features.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-tech-preview-features_{context}"]
+= Generally Available and Technology Preview features
+
+Features which are Generally Available (GA) are fully supported and are suitable for production use. Technology Preview (TP) features are experimental features and are not intended for production use. See the link:https://access.redhat.com/support/offerings/techpreview[Technology Preview scope of support on the Red Hat Customer Portal] for more information about TP features.
+
+The following table provides information about which {ServerlessProductName} features are GA and which are TP:
+
+// OCP table
+ifdef::openshift-enterprise[]
+.Generally Available and Technology Preview features tracker
+[cols="3,1,1,1,1",options="header"]
+|====
+|Feature |1.18|1.19|1.20|1.21
+
+|`kn func emit` (`kn func invoke` in 1.21+)
+|TP
+|TP
+|TP
+|TP
+
+|Service Mesh mTLS
+|GA
+|GA
+|GA
+|GA
+
+|`kn func` TypeScript templates
+|TP
+|TP
+|TP
+|TP
+
+|`kn func` Rust templates
+|TP
+|TP
+|TP
+|TP
+
+|`emptyDir` volumes
+|GA
+|GA
+|GA
+|GA
+
+|HTTPS redirection
+|-
+|GA
+|GA
+|GA
+
+|Kafka broker
+|-
+|-
+|TP
+|TP
+
+|====
+endif::[]
+
+// OSD table
+ifdef::openshift-dedicated[]
+.Generally Available and Technology Preview features tracker
+[cols="3,1,1",options="header"]
+|====
+|Feature |1.20|1.21
+
+|`kn func emit` (`kn func invoke` in 1.21+)
+|TP
+|TP
+
+|Service Mesh mTLS
+|GA
+|GA
+
+|`kn func` TypeScript templates
+|TP
+|TP
+
+|`kn func` Rust templates
+|TP
+|TP
+
+|`emptyDir` volumes
+|GA
+|GA
+
+|HTTPS redirection
+|GA
+|GA
+
+|Kafka broker
```

Comment:
we also have the `KafkaSink` as TP on the 1.21 version (not sure if that's than included on a different PR, but just saying here to make sure that's not lost)

Reply:
Added 🙂 thanks!